docs/source/find_data/organizations/eri-u-tokyo.md

```diff
@@ -73,7 +73,7 @@ In addition to the original analog seismograms, ERI maintains extensive archives
 **ERI**  
 Additional information about the microfilmed seismogram archives at the ERI can be obtained from the
 [Historical Seismogram Search System](http://wwweic.eri.u-tokyo.ac.jp/susu/index_en.html).
-
```

Comment:
The correct link for the Historical Seismogram Search System is:

http://wwweic.eri.u-tokyo.ac.jp/susu/en/index.html

Please update.